apps/mobile/src/app/wallet/wallet-settings/configure/[wallet]/[account].tsx

```diff
@@ -5,34 +5,29 @@ import { useSafeAreaInsets } from 'react-native-safe-area-context';
 import { getAvatarIcon } from '@/components/avatar-icon';
 import { AccountCard } from '@/components/wallet-settings/account-card';
 import { AccountNameModal } from '@/components/wallet-settings/account-name-modal';
-import { Account } from '@/state/accounts/accounts';
-import {
-  useAccountByIndex,
-  userRenamesAccount,
-  userTogglesHideAccount,
-} from '@/state/accounts/accounts.slice';
-import { makeAccountIdentifer, useAppDispatch } from '@/state/utils';
+import { Account, AccountLoader } from '@/store/accounts/accounts';
+import { userRenamesAccount, userTogglesHideAccount } from '@/store/accounts/accounts.write';
+import { makeAccountIdentifer, useAppDispatch } from '@/store/utils';
 import { BottomSheetModal } from '@gorhom/bottom-sheet';
 import { t } from '@lingui/macro';
 import { useTheme } from '@shopify/restyle';
 import { useLocalSearchParams, useNavigation } from 'expo-router';
+import { z } from 'zod';
 
 import { Box, Cell, Eye1ClosedIcon, HeadIcon, PassportIcon, Theme } from '@leather.io/ui/native';
 
-function ConfigureAccount({
-  fingerprint,
-  accountIndex,
-  account,
-}: {
+interface ConfigureAccountProps {
   fingerprint: string;
   accountIndex: number;
   account: Account;
-}) {
+}
+function ConfigureAccount({ fingerprint, accountIndex, account }: ConfigureAccountProps) {
   const { bottom } = useSafeAreaInsets();
   const theme = useTheme<Theme>();
   const accountNameModalRef = useRef<BottomSheetModal>(null);
   const dispatch = useAppDispatch();
   const navigation = useNavigation();
+
```

Comment:
I know it's not your code but often on mobile PRs now I see lint issues flagged like:
![Screenshot 2024-09-06 at 09 19 33](https://github.com/user-attachments/assets/311dd1a8-74be-4d21-b7ad-9bd6543ce09c)
![Screenshot 2024-09-06 at 09 19 43](https://github.com/user-attachments/assets/d6baa838-e5b1-4f7c-a900-72ba405f2c83)

Maybe we can make a team effort to fix them in PRs as we see them?